app/assets/images-loader.js

```diff
@@ -0,0 +1,4 @@
+require("file-loader?name=model-type-images/[name].svg!./model-type-images/account-active.svg");
```

Comment:
Something is breaking with the way you are loading and using the images.

If I browse to "/registration" it looks ok. When I browse to "/registration/wallet" and refresh, the images do not load
![image](https://user-images.githubusercontent.com/33128181/46615583-d485eb00-cb18-11e8-933d-e1e8481fead7.png)

Reply:
Urls registration/wallet and registration/account will lead to another pages, so the problem won't be actual.

Reply:
I'm curious as to how this is emerging. Have you found a reason?